.travis.yml

```diff
@@ -42,19 +40,19 @@ deploy:
   script: docs/publish.sh
   skip_cleanup: true
   on:
-    python: '3.5'
+    python: 3.8
```

Comment:
I think the idea is to test in the lower version not the latest.

Reply:
This task is not a test.  Here we are publishing docs.  If we can successfully publish our docs under Python 3.8 that this task should continue to work thru the [EOL date of v3.8](https://devguide.python.org/#branchstatus) which is 2024 instead of 9/2020 for v3.5.